README.md

```diff
@@ -177,6 +203,22 @@ Cache-Control: no-cache
 
 `````````````
 
+Possible Issues
+---------------
+
+You might experience very long startup times on some cloud hosted servers.  This might be because of a shortage
+of entropy as a result of no keyboard, or mouse:
+
+http://www.issihosts.com/haveged/
+
+To rectify, install the above software (Ubuntu instructions):
```

Comment:
Can the build process detect the need for `haveged` and recommend it to the user?  Perhaps a unit test that tries to get a random number and if it takes "too long", then the test fails with an error message that `haveged` needs to be installed.